homeassistant/components/dominos/services.yaml

```diff
@@ -0,0 +1,6 @@
+dominos_order:
```

Comment:
it shoudl just be `order` according to the docs: https://www.home-assistant.io/integrations/dominos/

and I think the code too: https://github.com/home-assistant/home-assistant/blob/a8aa3dcf7ed28fe7352b95a6756233f86d6a4aeb/homeassistant/components/dominos/__init__.py#L72

Reply:
Thanks, completely missed that one

Reply:
Fixed with 6a852676b79acc7a32b7aa357de8471f674cc21f